go/libkb/secret_store_file.go

```diff
@@ -98,8 +239,36 @@ func (s *SecretStoreFile) ClearSecret(username NormalizedUsername) error {
 	return nil
 }
 
+func (s *SecretStoreFile) clearSecretV2(username NormalizedUsername) error {
+	exists, err := FileExists(s.userpathV2(username))
```

Comment:
maybe this check should be on `noisepathV2` rather than `userpathV2`?  That seems more parallel.